bfpxe

```diff
@@ -149,7 +149,7 @@ fi
 # shellcheck disable=SC2013
 for i in $(cat /proc/cmdline); do
   # Network config.
-  if [ "$(echo "$i" | cut -c1-6)" = "bfnet=" || "$(echo "$i" | cut -c1-6)" = "bfmac=" ]; then
+  if [ "$(echo "$i" | cut -c1-6)" = "bfnet=" ] || ["$(echo "$i" | cut -c1-6)" = "bfmac=" ]; then
```

Comment:
The second condition needs a space in the beginning:

`["$(echo "$i" | cut -c1-6)" = "bfmac=" ]`

to

`[ "$(echo "$i" | cut -c1-6)" = "bfmac=" ]`

Reply:
Good catch.